deepchecks/vision/utils/image_properties.py

```diff
@@ -127,35 +127,75 @@ def get_dimension(img) -> int:
+    {'name': 'Aspect Ratio', 'method': aspect_ratio, 'output_type': 'numerical'},
+    {'name': 'Area', 'method': area, 'output_type': 'numerical'},
+    {'name': 'Brightness', 'method': brightness, 'output_type': 'numerical'},
+    {'name': 'RMS Contrast', 'method': rms_contrast, 'output_type': 'numerical'},
+    {'name': 'Mean Red Relative Intensity', 'method': mean_red_relative_intensity, 'output_type': 'numerical'},
+    {'name': 'Mean Green Relative Intensity', 'method': mean_green_relative_intensity, 'output_type': 'numerical'},
+    {'name': 'Mean Blue Relative Intensity', 'method': mean_blue_relative_intensity, 'output_type': 'numerical'}
 ]
 
 
-def validate_properties(properties):
+def validate_properties(properties: List[Dict[str, Any]]):
     """Validate structure of measurements."""
-    expected_keys = ['name', 'method', 'output_type']
-    output_types = ['discrete', 'continuous']
     if not isinstance(properties, list):
         raise DeepchecksValueError(
-            f'Expected properties to be a list, instead got {properties.__class__.__name__}')
+            'Expected properties to be a list, '
+            f'instead got {type(properties).__name__}'
+        )
+
     if len(properties) == 0:
         raise DeepchecksValueError('Properties list can\'t be empty')
-    for label_property in properties:
-        if not isinstance(label_property, dict) or any(
-                key not in label_property.keys() for key in expected_keys):
-            raise DeepchecksValueError(f'Property must be of type dict, and include keys {expected_keys}')
-        if label_property['output_type'] not in output_types:
-            raise DeepchecksValueError(f'Property field "output_type" must be one of {output_types}')
+
+    expected_keys = ('name', 'method', 'output_type')
+    output_types = ('categorical', 'numerical')
+
+    errors = []
+
+    for index, image_property in enumerate(properties):
+
+        if not isinstance(image_property, dict):
+            errors.append(
+                f'Item #{index}: property must be of type dict, '
+                f'and include keys {expected_keys}. Instead got {type(image_property).__name__}'
+            )
+            continue
+
+        property_name = image_property.get('name') or f'#{index}'
+        difference = sorted(set(expected_keys).difference(set(image_property.keys())))
+
+        if len(difference) > 0:
+            errors.append(
+                f'Property {property_name}: dictionary must include keys {expected_keys}. '
+                f'Next keys are missed {difference}'
+            )
+            continue
+
+        property_output_type = image_property['output_type']
+
+        if property_output_type not in output_types:
+            errors.append(
+                f'Property {property_name}: field "output_type" must be one of {output_types}, '
+                f'instead got {property_output_type}'
+            )
+            continue
```

Comment:
Add here a deprecation warning - accept "continuous" and "discrete" as we previously allowed them, but add a warning to stop using them (and add a filter in deprecation_warnings.py)

Reply:
added